ci/purge_expired_questionnaire_state.yaml

```diff
@@ -25,7 +25,7 @@ run:
             --region europe-west2 \
             --gcs-location gs://dataflow-templates/"${DATAFLOW_TEMPLATE_VERSION}"/Datastore_to_Datastore_Delete \
             --parameters \
-            datastoreReadGqlQuery="SELECT * FROM \`questionnaire-state\` WHERE submitted_at < $((`date --utc +%s` - EXPIRATION_TIME_OFFSET_IN_SECONDS))",\
+            datastoreReadGqlQuery="SELECT * FROM \`questionnaire-state\` WHERE submitted_at < $((UNIX_SECONDS(TIMESTAMP `date -utc`) - EXPIRATION_TIME_OFFSET_IN_SECONDS))",\
```

Comment:
Doesn't look like this is working. I get:
```bash
bash: line 15: UNIX_SECONDS(Tue Aug 10 11:05:59 UTC 2021) - EXPIRATION_TIME_OFFSET_IN_SECONDS: syntax error in expression (error token is "(Tue Aug 10 11:05:59 UTC 2021) - EXPIRATION_TIME_OFFSET_IN_SECONDS")
```

Reply:
You will need to pass a datetime to Datastore as we are not storing timestamp for questionnaire state. https://cloud.google.com/appengine/docs/standard/python/datastore/gqlreference?csw=1